src/app/public/modules/country-field/types/country.ts

```diff
@@ -0,0 +1,18 @@
+export interface SkyCountryFieldCountry {
+
+  /**
+   * The name of the country
```

Comment:
It's not a complete sentence, but I'd still end with a period here.

Reply:
Done